README.md

```diff
@@ -1,57 +1,138 @@
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/react-animate-text/master.svg
+[david-dm-url]:https://david-dm.org/moxystudio/react-animate-text
+[david-dm-image]:https://img.shields.io/david/moxystudio/react-animate-text.svg
+[david-dm-dev-url]:https://david-dm.org/moxystudio/react-animate-text?type=dev
+[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/react-animate-text.svg
 
-{package-description}
+A react component that animates text per word and/or per line.
 
 ## Installation
 
 ```sh
-$ npm install @moxy/{package-name}
+$ npm install @moxy/react-animate-text
 ```
 
 This library is written in modern JavaScript and is published in both CommonJS and ES module transpiled variants. If you target older browsers please make sure to transpile accordingly.
 
 ## Motivation
 
-{package-motivation}
+Many projects share the necessity of having animated text, so creating a component for that every time is a waste. This component offers a flexible solution to animate text that you can easily integrate and use in your project.
 
 ## Usage
 
-{package-usage-example}
+```js
+import React from 'react';
+import AnimateText from '@moxy/react-animate-text';
+
+const MyComponent = (props) => (
+    <div className="container">
+        <AnimateText initialDelay={ 0.5 } wordDelay={ 0.5 }>
+            Lorem ipsum dolor sit amet.
+        </AnimateText>
+    </div>
+);
+
+export default MyComponent;
+```
 
-## API
+The `AnimateText` component uses the [`@moxy/react-split-text`](https://github.com/moxystudio/react-split-text) to split the text.
 
-{package-api-description}
+To import the stylesheet, one can import it on the project's entry CSS file:
 
-#### {package-api-prop-example}
+```css
+@import "@moxy/react-animate-text/dist/index.css";
+```
 
-Type: `object`
-Required: `true`
+Or in the project's entry JavaScript file:
 
-The `{package-api-prop-example}` has the following shape:
 ```js
-{package-api-prop-example}: PropTypes.shape({
-    foo: PropTypes.string,
-    bar: PropTypes.arrayOf(PropTypes.object),
-}).isRequired,
+import "@moxy/react-animate-text/dist/index.css";
+```
+
+## API
+
+These are the props available in `@moxy/react-animate-text`.
+
+#### children
+
+Type: `string` | Required: `true`
+
+Text to be split and animated.
+
+#### separator
+
+Type: `string` | Required: `false` | Default: `non-breaking space`
+
+The pattern describing where each split should occur, just like the one from `String.prototype.split()`.
+
+#### className
+
+Type: `string` | Required: `false`
+
+A className to apply to the container.
+
+#### shouldShow
+
+Type: `boolean` | Required: `false` | Default: `true`
+
+Whether or not you want the text to be displayed.
+
+#### transitionDuration
+
+Type: `number` | Required: `false` | Default: `0.8`
+
+The transition duration in seconds for each animation.
+
+#### initialDelay
+
+Type: `number` | Required: `false` | Default: `0.3`
```

Comment:
This default value should be `0` as it's the initial animation, should leave this as vanilla as possible.